engine/src/test/java/io/camunda/zeebe/engine/state/instance/EventScopeInstanceStateTest.java

```diff
@@ -404,7 +404,7 @@ void shouldTriggerStartEventForNonExistingEventScope() {
     final EventTrigger eventTrigger = createEventTrigger();
 
     // when
-    triggerStartEvent(SCOPE_KEY, 456, eventTrigger);
+    triggerStartEvent(SCOPE_KEY, 456, eventTrigger, -1L);
```

Comment:
:wrench: We should verify in a test case that the process instance key of the event trigger is stored in the state.